### DISCUSSION TITLE
Urgent inquiries about the suitability of the project's operating system

### DISCUSSION THREAD

**Discussion by echo-lg** [fcitx/fcitx5#473] [Q&A]
I am solving the Chinese adaptation of Qt6 to the fcitx input framework on macOS, I want to ask whether the framework supports compiling and installing on macOS,

**Comment by wengxt** [comment]
mac is not a supported platform for fcitx  (yes it might can be built, but you can't do anything with it). There's no plan to support it since there's no one working on it. I don't own a mac device so it's impossible for me to work on it.
